app/frontend/src/components/forms/submission/ManageSubmissionUsers.vue

```diff
@@ -228,7 +229,7 @@ export default {
         this.isLoadingTable = false;
       }
     },
-    async modifyPermissions(userId, permissions) {
+    async modifyPermissions(userId, permissions, isAssigned) {
```

Comment:
I don't think(?) we need this new `isAssigned` flag on the endpoint?
The API side logic know if it's removing someone from a form because the permissions array is blank. Like how in \rbac\service.js it knows whether it's assigning a user to a form or removing them based on just body.permissions.

So the additional param all through the stack might be extraneous.